app/tests/components/Form/ReadOnlySubmissionWidget.test.tsx

```diff
@@ -0,0 +1,52 @@
+import FormTestRenderer from '../../utils/formTestRenderer';
+import { render, screen } from '@testing-library/react';
+import type { JSONSchema7 } from 'json-schema';
+
+const schema = {
+  title: 'Read only submission widget test',
+  type: 'object',
+  required: ['submissionDate'],
+  properties: {
+    submissionDate: {
+      title: 'On this date (YYYY-MM-DD)',
+      type: 'string',
+    },
+  },
+};
+
+const uiSchema = {
+  submissionDate: {
+    'ui:widget': 'ReadOnlySubmissionWidget',
+  },
+};
+
+const renderStaticLayout = (
+  schema: JSONSchema7,
+  uiSchema: JSONSchema7,
```

Comment:
```suggestion
  uiSchema: any,
```
That, or the uiSchema type exported by rjsf, if there is one. uiSchema isn't a json schema